### PR TITLE
Fix fly and fall damage

### PR DESCRIPTION
**Pull request by NexD4v**
https://github.com/Benedikt05/BetterAltay/issues/130

### CODE REVIEW

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -4372,7 +4372,9 @@ public function attack(EntityDamageEvent $source) : void{
 			and $source->getCause() !== EntityDamageEvent::CAUSE_VOID
 		){
 			$source->setCancelled();
-		}elseif($this->allowFlight and $source->getCause() === EntityDamageEvent::CAUSE_FALL){
+		}elseif(($this->isCreative() || $this->isSpectaror())
```
Comment (efor89) [inline src/pocketmine/Player.php:4375]: isSpectaror() to $this->isSpectator() plz

Comment (JoseLuisHD) [reply src/pocketmine/Player.php:4375]: Fly still needs another change from the .20-.40 to work in sync with mc

Comment (efor89) [reply src/pocketmine/Player.php:4375]: fly and falldamage is broken since the remove of the adventure settings

Comment (JoseLuisHD) [reply src/pocketmine/Player.php:4375]: https://github.com/Benedikt05/BetterAltay/commit/d016f393feea29938e6c20efcf1d2c167f37d595

Comment (efor89) [reply src/pocketmine/Player.php:4375]: it work if you remove $this->allowFlight in line 4376

Comment (efor89) [reply src/pocketmine/Player.php:4375]: but the rest is perfekt jose thx for the fast update

Comment (JoseLuisHD) [reply src/pocketmine/Player.php:4375]: The isSpectator function is not written correctly

**Pull request closed without merging**